pkg/apis/core/v1alpha1/defaults.go

```diff
@@ -163,6 +163,9 @@ func SetDefaults_Worker(obj *Worker) {
 	if obj.MaxUnavailable == nil {
 		obj.MaxUnavailable = &DefaultWorkerMaxUnavailable
 	}
+	if obj.Cri == nil {
+		obj.Cri = &DefaultWorkerCri
+	}
```

Comment:
is this necessary?

Reply:
After the discussion i will remove all true defaults.
It will be a change triggered only by users and explicitly.
Also updating the GEP.